marconi/test/Spec/Utxo.hs

```diff
@@ -0,0 +1,115 @@
+{-# LANGUAGE GADTs             #-}
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE NamedFieldPuns    #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PatternSynonyms   #-}
+{-# LANGUAGE TemplateHaskell   #-}
+{-# OPTIONS_GHC -fno-warn-incomplete-patterns #-}
+
+module Spec.Utxo (tests) where
+
+import Control.Lens (folded)
+import Control.Lens.Operators ((^.), (^..))
+import Control.Monad (forM)
+import Control.Monad.IO.Class (liftIO)
+import Data.List (nub)
+import Data.List.NonEmpty (nonEmpty, toList)
+import Data.Maybe (catMaybes, fromJust)
+import Database.SQLite.Simple qualified as SQLite
+
+import Hedgehog (Gen, Property, diff, forAll, property, (===))
+import Hedgehog qualified
+import Hedgehog.Gen qualified as Gen
+import Hedgehog.Range qualified as Range
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.Hedgehog (testPropertyNamed)
+
+import Cardano.Api qualified as C
+import Gen.Cardano.Api.Typed qualified as CGen
+import Marconi.Index.Utxo qualified as Utxo
+import Marconi.Indexers (uTxo, uTxoEvents)
+import Marconi.Types (CurrentEra, TargetAddresses)
+import RewindableIndex.Index.VSplit qualified as Ix
+
+tests :: TestTree
```

Comment:
Aren't these things tested by the previous tests? The ones that use a real indexer to store and query events?